test/test_prototype_transforms_consistency.py

```diff
@@ -655,37 +655,58 @@ def test_call_consistency(config, args_kwargs):
+            (prototype_transforms.ColorJitter, ArgsKwargs(brightness=None, contrast=None, saturation=None, hue=None)),
+            (prototype_transforms.ElasticTransform, ArgsKwargs(alpha=[15.3, 27.2], sigma=[2.5, 3.9], size=[17, 31])),
+            (prototype_transforms.GaussianBlur, ArgsKwargs(0.3, 1.4)),
+            (
+                prototype_transforms.RandomAffine,
+                ArgsKwargs(degrees=[-20.0, 10.0], translate=None, scale_ranges=None, shears=None, img_size=[15, 29]),
+            ),
+            (prototype_transforms.RandomCrop, ArgsKwargs(make_image(size=(61, 47)), output_size=(19, 25))),
+            (prototype_transforms.RandomPerspective, ArgsKwargs(23, 17, 0.5)),
+            (prototype_transforms.RandomRotation, ArgsKwargs(degrees=[-20.0, 10.0])),
+            (prototype_transforms.AutoAugment, ArgsKwargs(5)),
+        ]
+    ],
 )
-def test_get_params_alias(config):
+
+
+@get_paramsl_parametrization
+def test_get_params_alias(config, get_params_args_kwargs):
     assert config.prototype_cls.get_params is config.legacy_cls.get_params
 
+    if not config.args_kwargs:
+        return
+    args, kwargs = config.args_kwargs[0]
+    legacy_transform = config.legacy_cls(*args, **kwargs)
+    prototype_transform = config.prototype_cls(*args, **kwargs)
 
-@pytest.mark.parametrize(
-    ("transform_cls", "args_kwargs"),
-    [
-        (prototype_transforms.RandomResizedCrop, ArgsKwargs(make_image(), scale=[0.3, 0.7], ratio=[0.5, 1.5])),
-        (prototype_transforms.RandomErasing, ArgsKwargs(make_image(), scale=(0.3, 0.7), ratio=(0.5, 1.5))),
-        (prototype_transforms.ColorJitter, ArgsKwargs(brightness=None, contrast=None, saturation=None, hue=None)),
-        (prototype_transforms.ElasticTransform, ArgsKwargs(alpha=[15.3, 27.2], sigma=[2.5, 3.9], size=[17, 31])),
-        (prototype_transforms.GaussianBlur, ArgsKwargs(0.3, 1.4)),
-        (
-            prototype_transforms.RandomAffine,
-            ArgsKwargs(degrees=[-20.0, 10.0], translate=None, scale_ranges=None, shears=None, img_size=[15, 29]),
-        ),
-        (prototype_transforms.RandomCrop, ArgsKwargs(make_image(size=(61, 47)), output_size=(19, 25))),
-        (prototype_transforms.RandomPerspective, ArgsKwargs(23, 17, 0.5)),
-        (prototype_transforms.RandomRotation, ArgsKwargs(degrees=[-20.0, 10.0])),
-        (prototype_transforms.AutoAugment, ArgsKwargs(5)),
-    ],
-)
-def test_get_params_jit(transform_cls, args_kwargs):
-    args, kwargs = args_kwargs
+    assert prototype_transform.get_params is legacy_transform.get_params
+
+
+@get_paramsl_parametrization
+def test_get_params_jit(config, get_params_args_kwargs):
+    get_params_args, get_params_kwargs = get_params_args_kwargs
+
+    torch.jit.script(config.prototype_cls.get_params)(*get_params_args, **get_params_kwargs)
+
+    if not config.args_kwargs:
+        return
+    args, kwargs = config.args_kwargs[0]
+    transform = config.prototype_cls(*args, **kwargs)
```

Comment:
Why are we using `get_params_args_kwargs` for one of them and `config.args_kwargs[0]` for the other?

Reply:
`get_params_args_kwargs` is coming from the parametrization and only includes a single `ArgsKwargs` to test calling `get_params`. `config.args_kwargs` is a list of `ArgsKwargs` that is used to instantiate the transform and is coming from the `CONSISTENCY_CONFIGS`. Since `get_params` is a `@staticmethod`, we don't need to care about the actual parameters the transform is instantiated with. Thus, we just take the first available to get an arbitrary instance, and call `get_params` on that.